priv/gateway-conf/priv

```diff
@@ -0,0 +1 @@
+wFpTyodksFmh1zI5nbDxFUPs5sFvIKF7DBVaTWRj4Gc=
```

Comment:
What is this key for? Trying to make sure we won't have any kind of default secret by accident.